libraries/core_libs/network/src/network.cpp

```diff
@@ -13,32 +13,33 @@
 
 namespace taraxa {
 
-Network::Network(FullNodeConfig const &config, const h256 &genesis_hash,
+Network::Network(const FullNodeConfig &config, const h256 &genesis_hash,
                  dev::p2p::Host::CapabilitiesFactory construct_capabilities,
                  std::filesystem::path const &network_file_path, dev::KeyPair const &key, std::shared_ptr<DbStorage> db,
                  std::shared_ptr<PbftManager> pbft_mgr, std::shared_ptr<PbftChain> pbft_chain,
                  std::shared_ptr<VoteManager> vote_mgr, std::shared_ptr<NextVotesManager> next_votes_mgr,
                  std::shared_ptr<DagManager> dag_mgr, std::shared_ptr<TransactionManager> trx_mgr)
-    : kConf(config), tp_(config.network.network_num_threads, false) {
+    : tp_(config.network.network_num_threads, false) {
   auto const &node_addr = key.address();
   LOG_OBJECTS_CREATE("NETWORK");
-  LOG(log_nf_) << "Read Network Config: " << std::endl << kConf.network << std::endl;
+  LOG(log_nf_) << "Read Network Config: " << std::endl << config.network << std::endl;
```

Comment:
why did you renamed constant member?

Reply:
it is not a member anymore, it is ctor argument